api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -103,8 +104,15 @@ public void syncBillingProjectStatus() {
           default:
             break;
         }
+      } catch (NotFoundException e) {
+        log.log(Level.WARNING, "Get BillingProjectStatus call failed for " + entry.getCreationTime() + ". Project not found.");
```

Comment:
I would also ensure you log the original exception, i.e. as the third parameter to `log()`